website/src/components/BackButton.tsx

```diff
@@ -6,10 +6,11 @@ interface BackButtonProps {
     href: string
     text: string
     bold?: boolean
+    dark?: boolean
 }
 
-export const BackButton: FunctionComponent<BackButtonProps> = ({ href, text, bold }) => (
-    <Link className="btn p-0 text-uppercase mb-3 font-weight-normal" to={href}>
+export const BackButton: FunctionComponent<BackButtonProps> = ({ href, text, bold, dark }) => (
```

Comment:
I would love to get @bretthayes's opinion here, but it would be great imo if we could move away from toggling so many styles with props. I realize that I did this here as well by introducing `bold`, but would it make sense to have different versions of `BackButton` to clearly express intention? `BackButtonBold`, `BackButtonLight`, `BackButton` for example, to handle the three different ways we are currently using it.

Reply:
I should've made a comment of this, but eventually all the Case Study pages will change to the new styles introduced by this branch, so we won't need the light font color on the back button then. I was thinking of this as an interim until that change happens. Would you prefer a whole other `BackButtonLight` component until that change happens? @katjuell @bretthayes

Reply:
Yeah. Then we will just remove it when we don't need it.

Reply:
Yeah, I agree with @katjuell! If there are too many styles as props, components become too complex. But if it's for a few changes like changing a `fontWeight` prop as a variation, I think it feels more simplistic to add a few lines of logic vs duplicating markup. But if we are planning to throw away this anyway then I think what you're doing is cool in the interim.